File: scraps-strona/src/types/product.ts
export type Product = {
    id: number;
    images: string[];
    name: string;
    description: string;
    price: number;
    longDescription: string;
    specifications: ProductSpecification[];
    category: string;
    stock: number;
}

export type ProductSpecification = {
  name: string
  value: string
}

File: scraps-backend/app/domain/model/product.py
from dataclasses import dataclass
from decimal import Decimal
from typing import List

@dataclass
class Product:
    id: int
    name: str
    price: Decimal
    description: str
    long_description: str | None
    category: str
    stock: int
    images: list[str]
    specifications: list[dict]
    
